docs/config.rst

```diff
@@ -110,7 +110,20 @@ Core configuration
     The original MPD server only supports 10000 tracks in the tracklist. Some
     MPD clients will crash if this limit is exceeded.
 
-
+.. confval:: core/restore_state
```

Comment:
'We' as in the project / the people the developing it. But yeah naming things is hard :-)